data/assets/hoverCircle.asset

```diff
@@ -0,0 +1,25 @@
+local id = "HoverCircle"
+
+local circle = {
+  Identifier = id,
+  Type = "ScreenSpaceImageLocal",
+  Name = "HoverCircle",
+  FaceCamera = false,
+  UseRadiusAzimuthElevation = false,
+  RadiusAzimuthElevation = {1.0, 0.0, 0.0}, -- use for dome
+  Scale = 0.025,
+  Enabled = false,
+  TexturePath = "${ASSETS}/circle.png",
```

Comment:
This texture should probably be a synced resource?

Reply:
Alex had the same comment. Don't know how to fix that though :)